typescript/scripting-api/index.d.ts

```diff
@@ -32,15 +51,7 @@ declare namespace WoT {
      */
     export interface ThingFilter {
         /**
-         * The method field represents the discovery type that should be used in the discovery process. The possible values are defined by the DiscoveryMethod enumeration that can be extended by string values defined by solutions (with no guarantee of interoperability). 
-         */
-        method?: DiscoveryMethod | string; // default value "any", DOMString
-        /**
-         * The url field represents additional information for the discovery method, such as the URL of the target entity serving the discovery request, such as a Thing Directory or a Thing.
-         */
-        url?: string;
-        /**
-         * The query field represents a query string accepted by the implementation, for instance a SPARQL query. 
+         * The query field represents a query string accepted by the implementation, for instance a SPARQL query.
          */
         query?: string;
         /**
```

Comment:
Thank you :) I wasn't entirely sure about this one because in the source HTML file `query` is just commented out, but I suppose removing makes more sense to perform a proper alignment.

Reply:
Maybe it comes back but for the time being I think we should remove it.